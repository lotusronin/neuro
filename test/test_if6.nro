import stdio;

fn main() : u32 {
    var1 := 4;
    var2 := 5;
    if(var1 > 1) {
        stdio::puts("first if passed");
        if(var2 > 1) {
            stdio::puts("second if passed");
            if(var1 < var2) {
                stdio::puts("third if passed");
            }
        }
    }
    return 0;
}
