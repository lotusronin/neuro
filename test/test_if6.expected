first if passed
second if passed
third if passed
